src/auto-complete.ts

```diff
@@ -17,24 +18,33 @@ export class NguiAutoComplete {
         // ...
     }
 
-    public filter(list: any[], keyword: string, matchFormatted: boolean, accentInsensitive: boolean) {
-        return accentInsensitive
-            ? list.filter(
-                (el) => {
-                    const objStr = matchFormatted ? this.getFormattedListItem(el).toLowerCase() : JSON.stringify(el).toLowerCase();
-                    keyword = keyword.toLowerCase();
+    public filter(list: any[], keyword: string, matchFormatted: boolean, accentInsensitive: boolean, filters: AutoCompleteFilter[]) {
```

Comment:
@ctrahhuk This functionality will also be used for our Google Places integration (i.e. US Cities only). Will it work with their filtering mechanism as well? What I mean is, for Google Places we need to pass an additional filtering parameter on the request itself and we won't be filtering ourselves the list of objects that come back.